README.md

```diff
@@ -178,22 +192,88 @@ podTemplate(containers: [
 }
 ```
 
+or
+
+```groovy
+podTemplate(yaml:'''\
+    apiVersion: v1
+    kind: Pod
+    spec:
+      containers:
+      - name: maven
+        image: maven:3.8.1-jdk-8
+        command:
+        - sleep
+        args:
+        - 99d
+      - name: golang
+        image: golang:1.16.5
+        command:
+        - sleep
+        args:
+        - 99d
+'''.stripIndent()) {
```

Comment:
Do you actually need use `stripIndent`? I think it could be dropped if you just also omitted the `\` on the initial line.

```console
$ (echo; echo '  foo: 1'; echo '  bar: 2') | yq e -P
foo: 1
bar: 2
```

Reply:
Seems to work indeed.